apps/arweave/src/ar_mining_server.erl

```diff
@@ -703,6 +705,9 @@ prepare_solution(steps, Candidate, Solution) ->
 							Solution#mining_solution{ steps = Steps })
 			end;
 		false ->
+			ar_events:send(solution, {stale,
```

Comment:
What's the significance of stale vs. rejected? Or maybe it's historical? (I see the rejected and stale event types already existed)

Reply:
We introduced this separation for the pool (ar_pool is relying on these events)